roles/openshift_gcp/tasks/docker_storage_setup.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: create the docker-storage-setup config file
+  template: src=docker-storage-setup.j2 dest=/etc/sysconfig/docker-storage-setup owner=root group=root mode=0644
```

Comment:
This lives under the container-runtime role.  I'd prefer if we consume the same one.  It will get included if you call https://github.com/openshift/openshift-ansible/blob/master/playbooks/openshift-node/private/image_prep.yml

This calls the `../../../prerequisites.yml`.  The settings needed are the following
```
container_runtime_docker_storage_type: overlay2
container_runtime_docker_storage_setup_device: /dev/xvdb
```

The role now has `docker_storage_setup_options` as well if you need to specify those.

Reply:
Removed